apkid/rules/apk/packers.yara

```diff
@@ -114,6 +115,23 @@ rule dexprotector : packer
     is_apk and 1 of ($encrptlib_*) and $encrptcustom
 }
 
+rule tencentbugly : packer
+{
+ /**
+ * Tencent Bugly v1.x/v2.x (MultiDex Packer)
+ **/
+ 
+  meta:
+    author = "Jasi2169"
+    description = "Tencent Bugly"
```

Comment:
Could you add a sample hash here, please. Also, is there a website for this? How is this different from the normal tencent packers?

Reply:
Isn't Bugly a crash report lib? https://mvnrepository.com/artifact/com.tencent.bugly/crashreport

Reply:
i am at wedding i will add extra stuff,i will send you sample caleb plz email me at jaspreetsinghplayer[at]gmail[dot]com i send you sample it is multidex packer includes tinker maybe that adds overlay,the developer of app will not talk about it but there is chinese forums which does shows i need to lock after wedding,see yeah